sample/app/src/main/java/com/example/activitystarter/fragment/TabbedPlaceholderFragment.java

```diff
@@ -11,8 +10,10 @@
 
 import activitystarter.ActivityStarter;
 import activitystarter.Arg;
+import androidx.fragment.app.Fragment;
```

Comment:
We should also have tests for older fragments

Reply:
Do you mean `android.app.Fragment`? That one is deprecated. Or do you mean the support v4 Fragment?

Reply:
I mean both. A lot of things are deprecated but still in common use.

Reply:
Alright, thanks. I'll add both of them back in.

Reply:
I tried to add them both in but support v4 and the androidx packages give some conflicts I haven't been able to resolve.